libs/net/dal/Services/ReportService.cs

```diff
@@ -61,14 +63,24 @@ public IEnumerable<Report> FindAll()
     /// <param name="report"></param>
     /// <returns></returns>
     /// <exception cref="Exception"></exception>
-    public async Task<IEnumerable<Content>> FindContentWithElasticsearchAsync(Report report)
+    public async Task<IEnumerable<API.Areas.Services.Models.Content.ContentModel>> FindContentWithElasticsearchAsync(Report report)
     {
-        var response = await _client.SearchAsync<Content>(s =>
+        var response = await _client.SearchAsync<API.Areas.Services.Models.Content.ContentModel>(s =>
         {
+            var json = report.Filter.ToJson();
             var result = s
                 .Pretty()
                 .Index(_elasticOptions.PublishedIndex) // TODO: Switch to unpublished if the report settings require it.
-                .Query(q => q.Raw(report.Filter.ToJson()));
+                .Query(q => q.Raw(json == "{}" ? "" : json));
```

Comment:
I really don't like the NEST library.  It doesn't allow us to send raw JSON requests to Elasticsearch.  I want to replace it soon so that we don't have the stupid issues the library is causing.